frontend/src/modules/displays/MultiStepWizard.tsx

```diff
@@ -155,6 +187,23 @@ const HFNCControls = (): JSX.Element => {
   );
 };
 
+/**
+ * some description
```

Comment:
create data from the data interface?

Reply:
It has not much purpose. I supposedly liked the way to wrap them in a function to create object rather repeating same object syntax for each condition. Its more of personal preference, We are free to update it anyway to simplify